### ISSUE TITLE
Documentation culture settings are ignored in the stylecop.json

### ISSUE THREAD

**Issue by artem-voropaev** [DotNetAnalyzers/StyleCopAnalyzers#3036]
I use next settings in the stylecop.json with StyleCop Analyzers 1.1.118

```json  
{
    "documentationRules": {
        "documentationCulture": "ru-RU"
    }
}
```
StyleCop warns me [SA1623] The property's documentation summary text should begin with: 'Gets or sets', but I use "documentationCulture": "ru-RU" and it should not warn me because my code is correct for this culture
```csharp    
/// <summary>
/// Получает или задает что-то.
/// </summary>
public string Foo { get; set; }
```

"Gets or sets" is "Получает или задает" in ru-RU (it always worked in Visual Studio with Settings.StyleCop file for .Net Framework projects)

My csproj
```xml
<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <TargetFramework>netstandard2.0</TargetFramework>
        <CodeAnalysisRuleSet>../main.ruleset</CodeAnalysisRuleSet>
        <DocumentationFile>$(OutputPath)\$(AssemblyName).xml</DocumentationFile>
    </PropertyGroup>

    <ItemGroup>
      <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" />
    </ItemGroup>

    <ItemGroup>
        <AdditionalFiles Include="../stylecop.json" />
    </ItemGroup>
    
</Project>

````

**Comment by artem-voropaev** [comment]
I't a copy of #2958. Why do yo close it? It's not solved.

**Comment by sharwell** [comment]
@artem-voropaev The issue is a bug in an external project (dotnet/roslyn#29014). We cannot resolve it here.

**Comment by artem-voropaev** [comment]
but my sdk version is not 2.1.300

**Comment by sharwell** [comment]
@artem-voropaev You'll need to ask in dotnet/coreclr#20979 which specific version of the SDK first contains the fix

**Comment by artem-voropaev** [comment]
Bug is still alive in  3.0.100 and Micrsoft closed and merged https://github.com/dotnet/coreclr/issues/20979. May be problem on your side?

**Comment by fubar-coder** [comment]
It works for me in VS (with locale `de-DE`), but I get the same error when I use JetBrains Rider.

**Comment by sharwell** [comment]
@fubar-coder You'll want to file that bug with JetBrains Rider to get it fixed.

**Comment by sharwell** [comment]
Duplicate of #2958
